### PR TITLE
feat: Create sharing invitations from Desktop app

### PR DESCRIPTION
**Pull request by shanesveller**
## Current behavior

The `Share` menu item under a Shared Service is currently inert.

## Proposed changes

Open an input window to provide the recipient's email address, and submit the invitation to the backend on confirmation. Display (mostly-unstyled) feedback below the form on a backend error.

## Checks

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [x] I accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified.

### CODE REVIEW

File: implementations/typescript/ockam/ockam_app/src/routes/invite/[outlet_addr]/CreateServiceInvitation.svelte [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    },
+  );
+  let list_projects = invoke("plugin:projects|list_projects").then(
+    (received_projects) => {
+      console.debug(received_projects);
+      projects = received_projects;
+      // TODO: more defensive code
+      project = projects.length > 0 ? projects[0] : null;
+    },
+  );
```
Comment (adrianbenavides) [inline implementations/typescript/ockam/ockam_app/src/routes/invite/[outlet_addr]/CreateServiceInvitation.svelte]: Non blocking but perhaps we should simplify the `create_service_invitation` payload and build the `CreateServiceInvitation` in the command handler like it's [done here](https://github.com/build-trust/ockam/blob/30357ee24f5ce433963997d568410addc712bcf9/implementations/rust/ockam/ockam_app/src/shared_service/tcp_outlet/create.rs#L64). Perhaps I'm missing something?

Comment (shanesveller) [reply implementations/typescript/ockam/ockam_app/src/routes/invite/[outlet_addr]/CreateServiceInvitation.svelte]: I'll explore that suggestion, thanks.

Comment (shanesveller) [reply implementations/typescript/ockam/ockam_app/src/routes/invite/[outlet_addr]/CreateServiceInvitation.svelte]: I did wind up including this redesign and I liked the outcome much better, thank you! Currently visible in c2602a01a2140697e552804139f3fabec1da1a10...d711d187e3e2daad358267bac5a981483d5ad6cd.

Comment (adrianbenavides) [reply implementations/typescript/ockam/ockam_app/src/routes/invite/[outlet_addr]/CreateServiceInvitation.svelte]: Nice! 💪

**Review by adrianbenavides — APPROVED** [review]
LGTM!

**Pull request merged**